samples/nrf9160/lwm2m_client/overlay-dtls-cid.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_LWM2M_CLIENT_UTILS_DTLS_CID=y
+CONFIG_LWM2M_TLS_SESSION_CACHING=y
```

Comment:
Should the TLS session caching be disabled here?

Reply:
At least according to documentation and kconfig.

Reply:
yes